string cmd
---
string processing_result_json
